Yona/Yona/nl.lproj/Localizable.strings

```diff
@@ -36,3 +36,11 @@
 "adddevice.user.infoText" = "Je hebt nog niet eerder ingelogd op dit apparaat. Ga naar je primaire device (waarschijnlijk je mobiel), open de Yona app, vervolgens naar instellingen en klik op ‘apparaat toevoegen’. Vul vervolgens onderstaande velden in.";
 
 "login.user.errorinfoText" = "Jouw account is geblokkeerd om veiligheidsredenen, gebruik de optie reset-toegangscode om jouw account te activeren.";
+
```

Comment:
Will add the keys of localization as per Android in next PR, So that we can have same keys for both iOS and Android.